app/src/main/java/org/schabi/newpipe/settings/VideoAudioSettingsFragment.java

```diff
@@ -21,6 +21,16 @@
     public void onCreate(@Nullable Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
 
+        String[] durationsDescriptions = getResources().getStringArray(R.array.seek_duration_description);
```

Comment:
Please add a comment above that you initialize the seek duration list

Reply:
done